src/css/Title.scss

```diff
@@ -1,117 +1,104 @@
-.title {
-  position: relative;
-  height: 36px;
-  width: 100%;
-
-  .justify-center {
-    justify-content: center;
-  }
-
-  > div > span,
-  div#lower-row > div {
-    width: inherit;
-    font-size: 13px;
-  }
-
-  div#lower-row > div.title-attempts {
-    width: 40px;
-    text-align: right;
-    vertical-align: bottom;
-  }
+@import "variables";
 
-  div#lower-row > div.title-category {
-    width: calc(100% - 80px);
-    padding-left: 40px;
-    display: inline-flex;
-    align-items: center;
-    height: 100%;
-  }
-
-  > div.meta-left > span,
-  div.meta-left > div#lower-row > div:not(.title-attempts) {
-    text-align: left;
-  }
-
-  div.meta-left > div#lower-row > div.title-category {
-    width: calc(100% - 40px);
-    padding: 0;
-    justify-content: baseline;
-  }
-
-  div.title-attempts {
-    position: absolute;
-    right: 6px;
-    bottom: 4px;
-  }
-  div.meta-two-lines > div#lower-row > div.title-category {
-    align-items: baseline;
-    vertical-align: top;
-    padding-top: 1px;
-  }
-
-  div.meta-two-lines > div#lower-row > div.title-attempts {
-    vertical-align: top;
-    padding-bottom: 0;
-    padding-top: 1px;
-  }
+$title-font-size: 13px;
+$attempts-width: 40px;
+$side-padding: 6px;
```

Comment:
This is the same as the column padding and it is used everywhere else as well. We should move this into the variables.scss

(Though this is part of #192 so it doesn't need to be done now)